web/components/ui/PasswordStrength.tsx

```
@@ -0,0 +1,127 @@
+          <div className={`flex items-center gap-1.5 ${metCriteria.hasNumber ? 'text-green-600 dark:text-green-400 font-bold' : ''}`}>
+             {metCriteria.hasNumber ? <Check size={12} strokeWidth={3} /> : <div className="w-3" />}
+             Number
+          </div>
+          <div className={`flex items-center gap-1.5 ${metCriteria.hasSpecial ? 'text-green-600 dark:text-green-400 font-bold' : ''}`}>
+             {metCriteria.hasSpecial ? <Check size={12} strokeWidth={3} /> : <div className="w-3" />}
+             Symbol
+          </div>
+      </div>
+
+      {/* Hidden live region for accessibility */}
+      <div className="sr-only" role="status" aria-live="polite">
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)